deploy/crds/org_v1_che_cr.yaml

```diff
@@ -56,6 +56,27 @@ spec:
     workspaceNamespaceDefault: ''
     # defines if user is able to specify namespace different from the default
     allowUserDefinedWorkspaceNamespaces: false
+    # Sets the server and workspaces exposure type. Possible values are "multi-host", "single-host", "default-host".
+    # Defaults to "multi-host" which creates a separate ingress (or route on OpenShift) for every requied
+    # endpoint.
+    # "single-host" creates a single ingress/route and exposes workspaces on subpaths. Please read the docs
+    # to learn about the limitations of this approach.
+    # "default-host" exposes che server on the host of the cluster. Please read the docs to learn about
+    # the limitations of this approach.
+    serverExposureStrategy: ''
+    # When the serverExposureStrategy is set to "single-host", the way server and and workspaces
+    # are exposed is further configured by this property. The possible values are "native" (which means
+    # that the server and workspaces are exposed using ingresses on K8s) or "gateway" where the server
+    # and workspaces are exposed using a custom gateway based on Traefik.
+    # "native" is only supported on Kubernetes, "gateway" is supported on both openshift and kubernetes.
```

Comment:
Since this property only makes sense for the raw kubernetes case (it has only 1 possible value on OpenShift), why not adding it under the `K8S` section ?